projects/packages/my-jetpack/_inc/components/product-cards-section/index.jsx

```diff
@@ -52,11 +49,7 @@ const ProductCardsSection = () => {
 
 				return (
 					<Col tagName="li" sm={ 4 } md={ 4 } lg={ 4 } key={ key }>
-						<Item
-							admin={ !! window?.myJetpackInitialState?.userIsAdmin }
-							productData={ productData[ key ] }
-							fetchingProductData={ fetchingProductData }
-						/>
+						<Item admin={ !! window?.myJetpackInitialState?.userIsAdmin } />
```

Comment:
I'm going to go ahead and merge this as-is for now. We are missing selectors/ reducers/ actions for any of the initial state object - I think this would be a nice improvement, but maybe good to refactor other similar instances at the same time.